### ISSUE TITLE
Speaker notes view can not be opened in a tab

### ISSUE THREAD

**Issue by iBobik** [hakimel/reveal.js#2002]
From a slides we can open speaker notes by S key. It will open new window, but it is not possible to open it in the tab.

**Comment by rugk** [comment]
Good idea. It would IMHO be the best solution for https://github.com/hakimel/reveal.js/issues/2027. You can then still drag the tab out of the current window to another screen, all browsers support that…

And the popup does not really make it easier, as it also just opens on the current screen. So you have to move the window to the other screen, anyway…
